docs/v1.md

```diff
@@ -59,6 +60,23 @@
 - has(id: `string`) : `boolean`
 - isEmpty() : `boolean`
 
+## Components
+- servers(): `Servers`
+- channels(): `Channels`
+- messages(): `Messages`
+- schemas(): `Schemas`
+- channelParameters(): `ChannelParameters`
+- serverVariables(): `ServerVariables`
+- operationTraits(): `OperationTraits`
+- messageTraits(): `MessageTraits`
+- correlationIds(): `CorrelationId[]`
```

Comment:
Should we create `CorrelationIds` model?

Reply:
I did at the very beginning, but then removed. The reason is that there is no collection nor map, but always an array, since correlationId doesn't have an Id to be used.

Reply:
But in this case you will have key of map.

Reply:
Oh thats right. In components we do. 👍 adding

Reply:
done